src/main/java/com/questhelper/helpers/quests/deathontheisle/DeathOnTheIsle.java

```diff
@@ -0,0 +1,734 @@
+		investigateMurder.addStep(and(investigatedCellarItems, not(inVillaCellar), not(interrogatedPatziAndAdala)), interrogatePatziAndAdala);
+		investigateMurder.addStep(and(inVillaCellar, investigatedCellarItems), leaveCellar);
+		investigateMurder.addStep(and(not(inVillaCellar), not(investigatedCellarItems)), enterTheCellarAgain);
+		investigateMurder.addStep(not(investigatedJug), investigateJug);
+		investigateMurder.addStep(not(investigatedSmallBoxInSouthRoom), investigateSmallBoxInSouthRoom);
+		investigateMurder.addStep(not(investigatedBrokenStoolInSouthRoom), investigateBrokenStoolInSouthRoom);
+		investigateMurder.addStep(not(investigatedWineStorageInEastRoom), investigateWineStorageInEastRoom);
+		investigateMurder.addStep(not(investigatedBrokenPotteryInEastRoom), investigateBrokenPotteryInEastRoom);
+		investigateMurder.addStep(not(investigatedLiviusInEastRoom), investigateLiviusInEastRoom);
+
+		/// 26
+		// TODO: the pickpocket steps can fail, do we prompt the user to retry until they succeed?
+		pickpocketAdala = new NpcStep(this, NpcID.ADALA_13823, new WorldPoint(1446, 2936, 0), "Pickpocket Adala until you find Wine labels.", wineLabels);
+		pickpocketCozyac = new NpcStep(this, NpcID.COZYAC_13828, new WorldPoint(1445, 2934, 0), "Pickpocket Cozyac until you find a threatening note.", threateningNote);
+		pickpocketPavo = new NpcStep(this, NpcID.PAVO_13832, new WorldPoint(1445, 2934, 0), "Pickpocket Pavo until you find a drinking flask.", drinkingFlask);
+		pickpocketXocotla = new NpcStep(this, NpcID.XOCOTLA_13830, new WorldPoint(1445, 2934, 0), "Pickpocket Xocotla until you find a shipping contract.", shippingContract);
+
+		inspectWineLabels = new DetailedQuestStep(this, "Inspect the Wine labels in your inventory", wineLabels.highlighted());
+		inspectThreateningNote = new DetailedQuestStep(this, "Inspect the Threatening note in your inventory", threateningNote.highlighted());
+		inspectDrinkingFlask = new DetailedQuestStep(this, "Inspect the Drinking flask in your inventory", drinkingFlask.highlighted());
+		inspectShippingContract = new DetailedQuestStep(this, "Inspect the Shipping contract in your inventory", shippingContract.highlighted());
+
+		returnToTheGuards2 = returnToTheGuards.copy();
+
+		// varbit 11233: 0 -> 1 when handing over clues to the guards
+
+		investigateMurder2 = new ConditionalStep(this, returnToTheGuards2);
+		investigateMurder2.addStep(not(inVilla), returnToButlerAndHeadInside);
+		investigateMurder2.addStep(and(pickpocketedItems, not(inspectedWineLabels)), inspectWineLabels);
+		investigateMurder2.addStep(and(pickpocketedItems, not(inspectedThreateningNote)), inspectThreateningNote);
+		investigateMurder2.addStep(and(pickpocketedItems, not(inspectedDrinkingFlask)), inspectDrinkingFlask);
+		investigateMurder2.addStep(and(pickpocketedItems, not(inspectedShippingContract)), inspectShippingContract);
```

Comment:
Fixed in [`14a4cac` (#1774)](https://github.com/Zoinkwiz/quest-helper/pull/1774/commits/14a4cac58e8f6a2e0281abdb473747fce3f75b5f)

We start by checking if a user is missing an item, and if they do, we tell them to pickpocket the relevant guest.
If the user has all items, it will fall through to checking if they've inspected the items. Which will then fall back to having them return to the guards.
I believe this should work as expected